provider/github/poster.go

```diff
@@ -32,17 +33,24 @@ const (
 
 // Poster posts comments as Pull Request Reviews.
 type Poster struct {
-	pool *ClientPool
-	conf ProviderConfig
+	pool           *ClientPool
+	conf           ProviderConfig
+	footerTemplate *template.Template
 }
 
 var _ lookout.Poster = &Poster{}
 
 // NewPoster creates a new poster for the GitHub API.
 func NewPoster(pool *ClientPool, conf ProviderConfig) *Poster {
+	tpl, err := newFooterTemplate(conf.CommentFooter)
+	if err != nil {
+		log.DefaultLogger.Warningf("no footer template being used: %s", err)
```

Comment:
I think we discussed via chat to not exit with error for an older template format. But thinking again about it, I've changed my mind and I think it goes more in line with other misconfigured settings to exit `lookoutd` if the template is invalid.
Otherwise someone that doesn't update the template may ignore this warning, and then wonder whey the comments don't have a footer anymore.

Reply:
¬¬ I hate you :P

ok, I'll do so ;)

Reply:
done by 5d7b11ed209f0551fa2839892fba91aa231de279